ches/emailHandler.ts

```diff
@@ -15,25 +18,59 @@ export const sendCreateRequestEmails = async (formData: PrivateCloudCreateReques
         .filter(Boolean)
         .map((item) => item?.email),
       from: 'Registry <PlatformServicesTeam@gov.bc.ca>',
-      subject: `${formData.name} deletion request received`,
+      subject: `${formData.name} provisioning request received`,
     });
-    // await chesService.send({
-    //   bodyType: "html",
-    //   body: email,
-    //   to: adminEmails,
-    //   from: "Registry <PlatformServicesTeam@gov.bc.ca>",
-    //   subject: `New Delete request in Registry waiting for your approval`,
-    // });
+    await chesService.send({
+      bodyType: 'html',
+      body: email,
+      to: adminEmails,
+      from: 'Registry <PlatformServicesTeam@gov.bc.ca>',
+      subject: `New Provisioning request in Registry waiting for your approval`,
+    });
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+export const sendRequestApprovalEmails = async (request: PrivateCloudRequestWithRequestedProject) => {
+  const email = render(RequestApproval({ request }), { pretty: true });
+  try {
+    await chesService.send({
+      bodyType: 'html',
+      body: email,
+      // For all project contacts. Sent when the project set deletion request is successfully submitted
+      to: [
+        request.requestedProject.projectOwner,
+        request.requestedProject.primaryTechnicalLead,
+        request.requestedProject.secondaryTechnicalLead,
+      ]
+        .filter(Boolean)
```

Comment:
Could we relocate this `filtering` process to the `ches service level`, as it seems to be duplicated in multiple places?